src/js/modules/infragistics.ui.editors.js

```diff
@@ -5327,8 +5317,9 @@
 		},
 		_getDisplayValue: function () { //Numeric Editor
 			var value = this._valueInput.val(),
-				decimalSeparator = this.options.decimalSeparator, decimalPoint = ".",
-				minDecimals = this.options.minDecimals, dataMode = this.options.dataMode,
+				decimalSeparator = this._getOptionOrRegionalValue("decimalSeparator"), decimalPoint = ".",
+				minDecimals = this._getOptionOrRegionalValue("minDecimals"), dataMode = this.options.dataMode,
+				displayFactor = this._getOptionOrRegionalValue("displayFactor"),

```

Comment:
is display factor part of numeric editor. As far as I remember it's part of Percent Editor. If so - it should be moved to percent editor logic. If no ignore my comment.

Reply:
Maybe that's correct, but that refactoring is not related to the current issue. I will leave it that way. The problem in the other comment is fixed.